src/commands/EjectCommand.js

```diff
@@ -0,0 +1,47 @@
+import Command from "../Command";
+import FileSystemUtilities from "../FileSystemUtilities";
+import async from "async";
+import path from "path";
+import userHome from "user-home";
+
+export default class EjectCommand extends Command {
+  initialize(callback) {
+    this.to = this.flags.to || userHome;
+    this.dry = this.flags.dry;
+
+    this.pkg = this.input[0];
+
+    if (this.dry) {
+      this.logger.info("The following directories would be moved");
+    }
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    if (this.pkg) {
+      const pkg = this.filteredPackages.filter((pkg) => pkg._package.name === this.pkg)[0];
```

Comment:
If a package has been passed on the command line, isn't it safe to assume `--scope` and `--ignore` (the flags that affect `this.filteredPackages`) are irrelevant?

`const pkg = this.packageGraph.get(this.pkg);` seems more appropriate here.